sklearn/ensemble/voting_classifier.py

```diff
@@ -161,11 +166,19 @@ def fit(self, X, y, sample_weight=None):
 
         self.estimators_ = Parallel(n_jobs=self.n_jobs)(
                 delayed(_parallel_fit_estimator)(clone(clf), X, transformed_y,
-                    sample_weight)
-                    for _, clf in self.estimators)
+                                                 sample_weight)
+                for _, clf in self.estimators if clf is not None)
 
         return self
 
+    @property
+    def _narej_weights(self):
```

Comment:
Confusing because there aren't any NaNs involved. How about `_weights_not_none`... or just remove this helper